did-web/src/lib.rs

```diff
@@ -133,11 +136,18 @@ impl DIDResolver for DIDWeb {
         match resp.error_for_status_ref() {
```

Comment:
I guess that could be a `if let Err(err) = ...`

Reply:
Fixed in 34643d1ceb1b68715dbd7152a4eaf0429061dd79